lib/bitcoin/BitcoinClient.ts

```diff
@@ -227,6 +229,23 @@ export default class BitcoinClient {
     };
   }
 
+  private async createWallet () {
+    const walletNameToUse = 'sidetreeDefaultWallet';

```

Comment:
Sanity: this name can be anything and subsequent calls to add address as watch-only does not need to reference a wallet by name etc?

Reply:
Correct